python/ml4ir/base/tests/test_base.py

```diff
@@ -0,0 +1,85 @@
+import unittest
+import tensorflow as tf
+import numpy as np
+import os
+import random
+import gc
+from argparse import Namespace
+from tensorflow.keras.metrics import Metric
+from tensorflow.keras.optimizers import Optimizer
+
+from ml4ir.base.model.relevance_model import RelevanceModel
+from ml4ir.base.model.losses.loss_base import RelevanceLossBase
+from ml4ir.base.model.scoring.scoring_model import ScorerBase, RelevanceScorer
+from ml4ir.base.model.scoring.interaction_model import InteractionModel, UnivariateInteractionModel
+from ml4ir.base.model.optimizers.optimizer import get_optimizer
+from ml4ir.base.io.local_io import LocalIO
+from ml4ir.base.io.logging_utils import setup_logging
+from ml4ir.base.features.feature_config import FeatureConfig
+from ml4ir.base.config.keys import ArchitectureKey
+from ml4ir.applications.ranking.model.ranking_model import RankingModel, LinearRankingModel
+from ml4ir.applications.ranking.model.losses import loss_factory
+from ml4ir.applications.ranking.model.metrics import metric_factory
+from ml4ir.applications.ranking.config.parse_args import get_args
+
+import warnings
+from typing import Union, List, Type
+
+warnings.filterwarnings("ignore")
+
+
+OUTPUT_DIR = "ml4ir/applications/ranking/tests/test_output"
+ROOT_DATA_DIR = "ml4ir/applications/ranking/tests/data"
+FEATURE_CONFIG_FNAME = "feature_config.yaml"
+
+
+class RelevanceTestBase(unittest.TestCase):
```

Comment:
add a docstring mentioning this is the base test class for the repo - `Relevance` seems to throw me off